docs/user/user/debugging/python.md

```diff
@@ -13,12 +13,25 @@ such as `pings.custom_ping.submit()`, will send it.
 
 ## Logging pings
 
-If the `GLEAN_LOG_PINGS` environment variable is set to `true`, pings are
-logged to the console on `DEBUG` level whenever they are submitted.
+Glean offers two options for logging from Python:
 
-Make sure that when you configure logging in your application, you set the level
-for the Python logging library to `DEBUG` or higher. Otherwise pings won't be
-logged even if `GLEAN_LOG_PINGS` is set to `true`.
+- **Simple logging API:** A simple API that only allows for setting the logging level, but includes all Glean log messages, including those from its networking subprocess. This is also the only mode in which [`GLEAN_LOG_PINGS`](../../reference/debug/logPings.md) can be used to display ping contents in the log.
+- **Flexible logging API:** Full use of the Python `logging` module, including its features for redirecting to files and custom handling of messages, but does not include messages from the networking subprocess about HTTP requests.
+
+### Simple logging API
+
+You can set the logging level for Glean log messages to `DEBUG` as follows:
+
+```python
+import logging
+from glean import Glean
+
+Glean.set_log_level(logging.DEBUG)
```

Comment:
As a consumer of the Glean Python package, I would find it useful to set the logging level for the simple logging API on Glean initialization. Either as a keyword argument to `initialize` or on the `Configuration` object. This way we would only have 1 call for Glean configuration instead of 2.

```python
Glean.initialize(
    application_id=__title__,
    application_version=__version__,
    upload_enabled=enable_telemetry is True,
    data_dir=Path(TemporaryDirectory().name),
    logging_level=logging.DEBUG,
    configuration=Configuration(server_endpoint=platform),
)
```

Or

```python
Glean.initialize(
    application_id=__title__,
    application_version=__version__,
    upload_enabled=enable_telemetry is True,
    data_dir=Path(TemporaryDirectory().name),
    configuration=Configuration(
        server_endpoint=platform, logging_level=logging.DEBUG
    ),
)
```

As opposed to:

```python
Glean.initialize(
    application_id=__title__,
    application_version=__version__,
    upload_enabled=enable_telemetry is True,
    data_dir=Path(TemporaryDirectory().name),
    configuration=Configuration(server_endpoint=platform),
)
Glean.set_log_level(logging.DEBUG)
```